load.go

```diff
@@ -69,6 +69,10 @@ func (om *OpenMock) populateBehaviors(mocks []*Mock) {
 	for i := range mocks {
 		m := mocks[i]
 		m.loadFile(om.TemplatesDir)
+		if m.Include != "" {
+			m = r.Behaviors[m.Include].patchedWith(*m)
+		}
+		r.Behaviors[m.Key] = m
```

Comment:
can we throw a warning here for duplicate keys?

Reply:
done!

but, there's a little bit more to do if we want to handle 'multiple keys' properly though, as is, we could have that multiple reply bodies problem

Reply:
yes, that can be done be in the next PRs. need to take care of that in their specific load functions.